pkgs/development/compilers/gcc/default.nix

```diff
@@ -48,19 +48,37 @@
 }:
 
 let
-  versions = import ./versions.nix;
```

Comment:
renamed to `gccVersions` so as not to conflict with `lib.versions` which was also used in this file.